site/content/docs/browser-checks/multiple-tabs.md

```diff
@@ -8,11 +8,11 @@ menu:
     parent: "Browser checks"
 ---
 
-Certain scenarios might requires us to handle new tab creation or multiple tabs at once. Playwright supports this case and, as a consequence, Checkly does as well.
+Certain scenarios might requires us to handle new tab creation or multiple tabs at once. Playwright Test supports this case and, as a consequence, Checkly does as well.
```

Comment:
s/might requires/might require

Reply:
I'd rewrite the first sentence: 

Certain scenarios may require us to create new or multiple tabs simultaneously.